src/main/java/org/broadinstitute/dsm/db/dto/settings/InstanceSettingsDto.java

```diff
@@ -63,6 +65,13 @@
         return Optional.of(gbfShippedTriggerDSSDelivered);
     }
 
+    public Optional<Boolean> isHasAddressTab() {
```

Comment:
either `is` or `has` will be enough, depends the context, here I would use `hasAddressTab`